.travis.yml

```diff
@@ -1,15 +1,12 @@
-sudo: required
+language: generic
 services:
   - docker
 env:
   global:
-    - DOCKER_COMPOSE_VERSION: 1.18.0
     - TRAVIS_CI: "true"
 before_install:
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
+  - cp theme/_config.yml.example theme/config.yml
+  - cp config.json.sample config.json
+  - echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
+  - docker-compose up -d
```

Comment:
Unless you've added unit tests, I think the thing here is that we're simply testing to see if the container builds, which is why I have the `docker-compose up` in the `script` tag.